src/ServiceControl.Audit/Auditing/AuditIngestor.cs

```diff
@@ -59,16 +102,7 @@ await Forward(stored, settings.AuditLogQueue)
             }
         }
 
-        public async Task Initialize(IDispatchMessages dispatcher)
-        {
-            this.dispatcher = dispatcher;
-            if (settings.ForwardAuditMessages)
```

Comment:
@timbussmann this check is now removed after the `Initialize` method has been moved to `AuditIngestion`.